### DISCUSSION TITLE
What's your experience with Pact.io?

### DISCUSSION THREAD

**Discussion by austrum** [MobileNativeFoundation/discussions#162] [General]
We're evaluating Pact.io for consumer driven contracts for our native apps. I can see a few examples of other companies leveraging this for native apps (e.g. DoorDash), but not a ton. Curious how many here leverage Pact and what your experience has been? What stumbling blocks did you it, if any, and how did you work past them?

**Comment by bootstraponline** [comment]
The [Canvas apps](https://github.com/instructure/canvas-android/commit/7be3327d9726800ab093cfcfaad847f0f5dd9daa) have some experience using pact. The support varies greatly depending on what language bindings are being used.

I think gRPC or similarly strongly typed APIs are preferable over REST + Pact.

**Comment by c1ira** [comment]
There’s https://pactflow.io/case-studies/
